Makefile

```diff
@@ -176,6 +176,20 @@ helm-install:
 	--set global.image.tag=zqd:$(ECR_VERSION) \
 	--set postgresql.persistence.enabled=$(PG_PERSIST)
 
+helm-install-with-aurora:
+	helm upgrade -i zsrv charts/zservice \
+	--set root.datauri=$(ZQD_DATA_URI) \
+	--set global.AWSRegion=us-east-2 \
+	--set global.image.repository=$(ZQD_ECR_HOST)/ \
+	--set global.image.tag=zqd:$(ECR_VERSION) \
+	--set global.postgres.addr=$$(aws rds describe-db-cluster-endpoints \
+		--db-cluster-identifier zq-test-aurora \
+		| jq -r ".DBClusterEndpoints[] | select(.EndpointType==\"WRITER\") | .Endpoint"):5432 \
```

Comment:
```suggestion
		--output text --query "DBClusterEndpoints[?EndpointType=='WRITER'] | [0].Endpoint"):5432 \
```

Reply:
Wow! Thanks!  You have mastered *ALL* the query languages. :-)